build-tools/automation/yaml-templates/run-integrated-regression-tests.yaml

```diff
@@ -26,7 +26,7 @@ steps:
   inputs:
     github_token: $(GitHub.Token)
     provisioning_script: $(System.DefaultWorkingDirectory)/provisionator/xamarin-android.csx
-    provisioning_extra_args: -vv
+    provisioning_extra_args: -vv -f
```

Comment:
Same as above, let's remove this as it would force re-installation of other test dependencies (Android SDK, Visual Studio, etc).